_kbarticles/2024-12-10-using-opensearch-dashboards.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Understanding OpenSearch Dashboards, logging and how to visualize your application logs"
+date: December 10, 2024
+excerpt: This article briefly explains what opensearch dashboards does, what types of logs are available in Dashboards, creating dashboards, and how to create log visualizations in Dashboards.
+redirect_from:
+  - /knowledge-base/2022-12-08-common-kibana-searches-on-clouddotgov/
+
+---
+
+## What does Dashboard do?
+
+Dashboard is a user interface that lets you search and visualize your application logs. Dashboard has a [user guide](https://opensearch.org/docs/latest/dashboards/) that explains more about how to use it and to create custom visualizations.
+
+## What are some types of logs?
+
+Cloud Foundry assigns a type to each log message depending on its origin. Application logs are assigned the APP log type. HTTP requests being routed to an app will produce the RTR log type. The various types of logs are listed in the documentation [here](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html#format).
+
+The log type is stored on logs in the `@source.type` field. So to query for application logs, you could use a filter of `@source.type: "APP"`.
+
+Cloud Foundry logs are often translated into field names in OpenSearch Dashboards using the log type as a prefix. For example, APP logs in Dashboards include fields like `app.name` for the application name.
```

Comment:
This isn't true. It seems to refer to how JSON logs were parsed into Kibana, which is different with flat_object in OpenSearch. No custom field names are created from logs in OpenSearch